app/controllers/users.controller.js

```diff
@@ -0,0 +1,20 @@
+const httpStatusCodes = require('http-status-codes');
+const { hashPassword } = require('../helpers/encryptions');
+
```

Comment:
Where are you valid if the user exists?

Reply:
It is directly validated with the sequilize schema, however in the next commit where validation is requested in the tests, I create an error handler to specify the error

Reply:
great